### DISCUSSION TITLE
Aggiornamento vecchia versione

### DISCUSSION THREAD

**Discussion by vinny74man** [indomus/forum#1503] [HUB: Home Assistant]
Ciao a tutti ho un installazione un po veccchiotta  la versione è la 0.99, fino ad ora tutto ok ma adesso devo ampliare con dei dispositivi modbus, ho provato la configurazione ma mi da vari errori leggendo un po in giro ho visto che altri hanno risolto con un aggiornamento, ho una versione core installata su un raspi PI3, posso eseguire direttamente l'upgrade ho meglio salvare la coonfigurazione e fare una nuova installazione, al momento comando dei dispositivi KNX sono circa una trentina, cosa sarebbe più sicuro?
Grazie

**Comment by snakuzzo** [comment]
Sei dietro di una vita. Puoi pure aggiornare all'ultima versione, ma dopo devi metterti a controllare i log e risolvere eventuali problemi.
Al posto tuo, se non hai tanta roba configurata, butterei tutto e ripartirei da capo con l'ultima versione di RaspOS ed installerei tutto con Docker

**Reply by vinny74man** [reply]
Perchè mi consigli docker io di solito uso sempre HomeAssistant Core con virtual-env, che benefici ci sono con l'installazione in docker

**Reply by snakuzzo** [reply]
https://indomus.it/guide/come-installare-docker-su-raspberry-pi-con-raspbian/

**Reply by KC-inDomus** [reply]
> Perchè mi consigli docker io di solito uso sempre HomeAssistant Core con virtual-env, che benefici ci sono con l'installazione in docker

gestione facilitata, performance, e molto altro
